### ISSUE TITLE
Bower.json error

### ISSUE THREAD

**Issue by SmallDigital** [cgkineo/adapt-pageNav#14]
I'm using the Authoring tool.  

Adapt authoring tool 0.10.5.
Adapt framework 5.11.0.

Unable to preview course when Page Nav component is added to a block. 

21 13:18:15 +00:00] ENOENT: no such file or directory, open '/Users/LeonardL/adapt_authoring/temp/60201f20acc458a6b564d7bb/adapt_framework/src/components/adapt-pageNav/bower.json'

Please find attached screenshots of errors:

<img width="942" alt="Screenshot 2021-03-13 at 13 32 06" src="https://user-images.githubusercontent.com/11190783/111031769-1067bd80-8401-11eb-81a5-5a524b91fe33.png">
<img width="1677" alt="Screenshot 2021-03-13 at 13 31 09" src="https://user-images.githubusercontent.com/11190783/111031777-16f63500-8401-11eb-9b8e-4562c39bbfc2.png">

Any pointers/assistance appreciated.

**Comment by SmallDigital** [comment]
@chris-steele @moloko @kirsty-hames  Hi - just wondering if you have come across the above issue? and if so do you have any pointers on how to resolve. If this is the wrong place to post issues like the above - let me know and I'll post it in the forum. Thanks.

**Comment by tomgreenfield** [comment]
> Laura Leonard
@PhotographyCamp_twitter
Mar 12 11:07
@squire:privacytools.io I've installed both adapt-footer and adapt-pageNav. - I got an error with the adapt-pageNav so I deleted it from the plugin management area in the tool. I tried to re-upload it and got an error: Plugin upload failed
Error: You already have this version of the plugin installed. Do I need to remove it from mongo as well or can I delete it from the src folder?

Judging by the Gitter message it looks like this might be an issue arising from the tinkering of the filesystem/database. Does the file referenced in the error message actually exist?

As @moloko points out, we use this plugin on a fair few authoring tool instances and I can't say this has cropped up before.

**Comment by SmallDigital** [comment]
> > Laura Leonard
> > @PhotographyCamp_twitter
> > Mar 12 11:07
> > @Squire:privacytools.io I've installed both adapt-footer and adapt-pageNav. - I got an error with the adapt-pageNav so I deleted it from the plugin management area in the tool. I tried to re-upload it and got an error: Plugin upload failed
> > Error: You already have this version of the plugin installed. Do I need to remove it from mongo as well or can I delete it from the src folder?
> 
> Judging by the Gitter message it looks like this might be an issue arising from the tinkering of the filesystem/database. Does the file referenced in the error message actually exist?
> 
> As @moloko points out, we use this plugin on a fair few authoring tool instances and I can't say this has cropped up before.

Hi  @moloko  @tomgreenfield  @taylortom - Thank you for coming back - I have checked the file structure. It appears to there.  Please find attached screenshots. Is there a way to reinstall the plugin from the command line rather than through the interface? Or do you have any other suggestions re: Database? Is there a way to remove the plugin and start again?  Or is it a case of me having to reinstall adapt? 

<img width="1468" alt="Screenshot 2021-03-15 at 15 31 38" src="https://user-images.githubusercontent.com/11190783/111179227-c408b300-85a3-11eb-9532-3867b13a3ded.png">
<img width="1278" alt="Screenshot 2021-03-15 at 15 33 41" src="https://user-images.githubusercontent.com/11190783/111179357-e8648f80-85a3-11eb-892d-261d2f63c452.png">

**Comment by moloko** [comment]
the folder name in your Finder screenshot is `adapt-pageNav-master` which is presumably why you're getting the error:
>  ENOENT: no such file or directory, open '/Users/LeonardL/adapt_authoring/temp/60201f20acc458a6b564d7bb/adapt_framework/src/components/adapt-pageNav/bower.json'

I have no idea if just renaming the folder would work or might have knock-on impacts but it might be worth a try?

**Comment by SmallDigital** [comment]
@moloko You are a genius!! That worked. Thank you so much. 

Like Tom said above - I did get an initial error so I tried to remove the plugin, then re-install it, and I obviously overwrote something between the versions! 

I have used the framework quite a bit in the past, but I thought I'd learn how to use the authoring tool; to be able to teach my students how to create e-learning content using Adapt.
